go/tests.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 {%- endif %}
       with:
         version: {{ golangci_lint_version }}
-        args: -E=gofmt --timeout=30m0s
+        args: --timeout=30m0s
```

Comment:
As the error, `-E is used to enable linters` and gofmt is not a linter. We are removing the format here, we should not dot it. Instead let's understand the  right replace for it.

Seems we have here:
`golangci-lint run -E=goimports` or either:
`golangci-lint run -E=gofumpt`

I'm not too familiar with the it, but let's try to understand it looking at the docs:

Seems to use goimports we will need to define the rules,  https://golangci.github.io/legacy-v1-doc/usage/linters/#goimports

I think we can try to keep the behavior for https://golangci.github.io/legacy-v1-doc/usage/linters/#gofmt
changing the cmd to support   `gofmt -l .`

Reply:
ahh, you said it here:
>> To resolve this, I removed -E=gofmt and added gofmt as a separate step to isolate formatting from linting. This way, if cmd format fails, it fails independently. I'm open to other approaches though, looks like using gofmt -l could also work. So, do you think it makes sense? Do you have any ideas?